### DISCUSSION TITLE
Result of Deployment Frequency as Yearly

### DISCUSSION THREAD

**Discussion by bruno561** [GoogleCloudPlatform/fourkeys#436] [Q&A]
I have frequent deployments (more than 5 per day) and a period of 5 days per week (only weekends which I don't have) and the Deployment Frequency result shows up as Yearly. What am I doing wrong?

![image](https://user-images.githubusercontent.com/65369082/232495854-2fefb778-0349-47a2-9637-4298bf035122.png)

**Comment by colinfrei** [comment]
The dashboard seems to always take data from the last three months for the deployment frequency, regardless of the timespan you select at the top right. Do you have enough data to have at least one deployment a month for the last three months?
